app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Api::V1::ItemsController < ApplicationController
+  before_action :set_item, only: %i[show update destroy]
+
+  # GET /api/v1/items
+  def index
+    @items = Item.all.includes(:reservations)
+
+    render json: @items
+  end
+
+  # GET /api/v1/items/1
+  def show
+    render json: @item
+  end
+
+  # POST /api/v1/items
+  def create
+    @item = Item.new(item_params)
+
+    if @item.save
+      render json: @item, status: :created, location: api_v1_item_url(@item)
+    else
+      render json: @item.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /api/v1/items/1
+  def update
+    if @item.update(item_params)
+      render json: @item
+    else
+      render json: @item.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /api/v1/items/1
+  def destroy
+    def destroy
+      if @item.destroy
+        render json: { message: 'Item deleted successfully.' }
+      else
+        render json: { message: 'Failed to delete item.' }, status: :unprocessable_entity
+      end
+    end
+  end
```

Comment:
- [ ] You are nesting this method (which means, a method _inside_ another method.

Kindly, unnest the method so it's consistent.  The `destroy` method should look like this:

```ruby
def destroy
    if @item.destroy
      render json: { message: 'Item deleted successfully.' }
    else
      render json: { message: 'Failed to delete item.' }, status: :unprocessable_entity
    end
end
```